appveyor.approval.yml

```diff
@@ -0,0 +1,18 @@
+version: '{build}'
+
+environment:
+    PATH: 'C:\Ruby200\bin;%PATH%'
+
+install:
+    - cmd: bundle install
+
+build_script:
+    - cmd: build.netstd.cmd approve

```

Comment:
Rather than having 2 appveyor files, could we configure an environment variable in the web UI which is used here? The env var could be omitted for the regular project, and set to "approve" for the approval project.